data/governments.txt

```diff
@@ -1052,6 +1163,10 @@ government "Uninhabited"
 	"bribe" 0
 	"fine" 0
 
+government "vyu-Rulei"
+	swizzle 0
+	color 1 1 1
```

Comment:
```suggestion
	color 1 1 1
	"player reputation" 0
```
I know it is defaulted, but we always give a value for it

Reply:
I don't think we do it was a mistake in the Hai PR lets not give useless info

Reply:
... but it is literally defined in any other gegno government?

Reply:
I guess its fine to add this but it should be erased everywhere later on

Reply:
It is the same with the end of some phrases btw, if it is not wanted, you might want to fix them too

Reply:
I believe this was left in at request. If we want to change this, it won't be in this PR since other things do this.